torch/nn/modules/loss.py

```diff
@@ -138,11 +139,6 @@ class NLLLoss(_WeightedLoss):
             \text{if reduction} = \text{`sum'.}
         \end{cases}
 
-    Can also be used for higher dimension inputs, such as 2D images, by providing
-    an input of size :math:`(minibatch, C, d_1, d_2, ..., d_K)` with :math:`K \geq 1`,
-    where :math:`K` is the number of dimensions, and a target of appropriate shape
-    (see below). In the case of images, it computes NLL loss per-pixel.
-
```

Comment:
Is this deletion intentional?

Reply:
Yes, I updated the `NLLLoss` docs slightly to be consistent with `CrossEntropyLoss` for this part. This part is mostly a repeat of the description of K-dimensional loss further up.